android-patches/trap-handler.h.patch

```diff
@@ -0,0 +1,26 @@
+--- trap-handler.h	2022-08-11 09:01:23.384000000 +0800
```

Comment:
Do you have any plan to upstream this patch to V8?

Reply:
I have not yet confirmed the availability of this patch on other systems, so for now I will use `patch` tool to move through the Android build process.

Reply:
OK then, let us mark this comment as resolved.